packages/docusaurus-preset-shiki-twoslash/index.js

```diff
@@ -51,39 +58,39 @@ function preset(context, pluginOptions) {
     return a
   }
 
-  // if a preset is found
-  // no need to search for plugins
-  if (context.siteConfig.presets.find(p => contains(presets, p))) {
-    context.siteConfig.presets = context.siteConfig.presets.map(preset => {
-      if (!contains(presets, preset)) {
-        return preset
-      }
-
-      const output = structure(preset)
-      const sections = ["docs", "blog", "pages"]
-      for (const section of sections) {
-        if (!output[1][section]) output[1][section] = {}
-        addTwoslash(output[1][section])
-      }
-
-      return output
-    })
-  } else {
-    context.siteConfig.plugins = context.siteConfig.plugins.map(plugin => {
-      // if the plugin is not supported
-      if (!contains(plugins, plugin)) {
-        return plugin
-      }
-
-      return addTwoslash(structure(plugin))
-    })
-  }
+  context.siteConfig.presets = context.siteConfig.presets.map(preset => {
```

Comment:
Review this with hide whitespace because all I did was stripping the if-else

Reply:
Aye, looks good - thanks